python/lib/dependabot/python/file_parser.rb

```diff
@@ -91,41 +91,109 @@ def package_manager
 
       sig { returns(Ecosystem::VersionManager) }
       def detected_package_manager
-        return PeotryPackageManager.new(detect_poetry_version) if poetry_lock && detect_poetry_version
+        setup_python_environment if Dependabot::Experiments.enabled?(:enable_file_parser_python_local)
+
+        return PeotryPackageManager.new(detect_poetry_version) if potery_files && detect_poetry_version
```

Comment:
typo: Poetry

Reply:
@abdulapopoola  thanks for noticing , fixed.